test_sets/PTU_no_certificate_exists_at_module_config.txt

```diff
@@ -0,0 +1,57 @@
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/033_ATF_P_TC_navi_app_PTU_retry_sequence_finish_ForceProtectedService_OFF_RPC_encrypted_true.lua Check_that_SDL_will_return_RPC_ACK_encrypt_false_when_PTU_fial_because_retry_sequence_finished
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/034_ATF_P_TC_navi_app_PTU_retry_sequence_finish_ForceProtectedService_OFF_Hybrid_encrypted_true.lua Check_that_SDL_will_return_Hybrid_ACK_encrypt_false_when_PTU_fial_because_retry_sequence_finished
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/035_ATF_P_TC_media_app_PTU_fail_ForceProtectedService_OFF_RPC_encrypted_true.lua Check_that_SDL_will_return_RPC_ACK_encrypt_false_when_PTU_fail
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/036_ATF_P_TC_comm_app_PTU_fail_ForceProtectedService_OFF_RPC_encrypted_true.lua Check_that_SDL_will_return_RPC_ACK_encrypt_false_when_PTU_fail
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/037_ATF_P_TC_default_app_PTU_fail_ForceProtectedService_OFF_RPC_encrypted_true.lua Check_that_SDL_will_return_RPC_ACK_encrypt_false_when_PTU_fail
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/038_ATF_P_TC_media_app_PTU_retry_sequence_finish_ForceProtectedService_OFF_RPC_encrypted_true.lua Check_that_SDL_will_return_RPC_ACK_encrypt_false_when_PTU_fial_because_retry_sequence_finished
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/039_ATF_P_TC_comm_app_PTU_retry_sequence_finish_ForceProtectedService_OFF_RPC_encrypted_true.lua Check_that_SDL_will_return_RPC_ACK_encrypt_false_when_PTU_fial_because_retry_sequence_finished
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/040_ATF_P_TC_default_app_PTU_retry_sequence_finish_ForceProtectedService_OFF_RPC_encrypted_true.lua Check_that_SDL_will_return_RPC_ACK_encrypt_false_when_PTU_fial_because_retry_sequence_finished
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/041_ATF_P_TC_navi_PTU_brings_certificate_Audio_ForceProtectedService_OFF.lua Check_that_SDL_will_reply_StartService_ACK_encrypted_true_PTU_brings_certificate
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/042_ATF_P_TC_navi_PTU_brings_certificate_Video_ForceProtectedService_OFF.lua Check_that_SDL_will_reply_StartService_ACK_encrypted_true_PTU_brings_certificate
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/043_ATF_P_TC_navi_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua Check_that_SDL_will_reply_StartService_ACK_encrypted_true_PTU_brings_certificate
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/044_ATF_P_TC_navi_PTU_brings_certificate_Hybrid_ForceProtectedService_OFF.lua Check_that_SDL_will_reply_StartService_ACK_encrypted_true_PTU_brings_certificate
```

Comment:
@istoimenova As I know Hybrid service (15) available on app registering. Do not need to start this service. Also it should be secure if RPC service becomes secure. Please clarify am I correctly remember.